extension/src/XLIFFDocument.ts

```diff
@@ -437,6 +448,7 @@ export class Target implements TargetInterface {
     textContent: string;
     state?: TargetState | null;
     translationToken?: TranslationToken;
+    stateQualifier?: StateQualifier;
```

Comment:
Don't know if we should use an enum for this.
Microsoft has invented their own values for this...
At least I've seen this one: `x-microsoft-exact-match`

We might start with a string  instead, otherwise would new values be blocking export....

Reply:
If we had a List we could always try to extend it :)